watch.go

```diff
@@ -146,9 +148,14 @@ func (w *FSWatcher) WatchDir(baseDir string) error {
 				}
 
 				path := event.Name
-
 				if ext := filepath.Ext(path); ext != ".mbtiles" {
```

Comment:
This line is now permissive for restarting file watcher; it triggers on everything that is not an mbtiles file, such as other files.

This needs to be limited to directory create / delete, and otherwise use `continue`.

Reply:
ok，I will optimize